conf/boards/lisa_mxs_1.0_nimble.makefile

```diff
@@ -0,0 +1,50 @@
+# Hey Emacs, this is a -*- makefile -*-
+#
+# lisa_mxs_1.0_nimble.makefile
+#
+# MXS 1.0 board setup for DelFly Nimble 
+# https://wiki.paparazziuav.org/wiki/Lisa/MXS_v1.0
+#
+
+BOARD=lisa_mxs
+BOARD_VERSION=1.0
+BOARD_CFG=\"boards/$(BOARD)_$(BOARD_VERSION).h\"
+
+ARCH=stm32
+ARCH_L=f4
+HARD_FLOAT=yes
+$(TARGET).ARCHDIR = $(ARCH)
+$(TARGET).LDSCRIPT=$(SRC_ARCH)/lisa-mx.ld
+
+# -----------------------------------------------------------------------
+
+HAS_LUFTBOOT ?= 0
+ifeq (,$(findstring $(HAS_LUFTBOOT),0 FALSE))
+$(TARGET).CFLAGS+=-DLUFTBOOT
+$(TARGET).LDFLAGS+=-Wl,-Ttext=0x8004000
+DFU_ADDR = 0x8004000
+DFU_PRODUCT = Lisa/Lia
+endif
+
+ASPIRIN_2_SPI_DEV ?= spi2
+ASPIRIN_2_SPI_SLAVE_IDX ?= SPI_SLAVE2
+
+SDLOGGER_DIRECT_SPI ?= spi1
+SDLOGGER_DIRECT_SPI_SLAVE ?= SPI_SLAVE1
+HS_LOG_SPI_DEV ?= spi1
+HS_LOG_SPI_SLAVE_IDX ?= SPI_SLAVE1 
+
+RADIO_CONTROL_LED  ?= none
+BARO_LED           ?= none
+AHRS_ALIGNER_LED   ?= none
+GPS_LED            ?= none
+SYS_TIME_LED       ?= 1
+LOGGER_LED         ?= 2
+
+FLASH_MODE ?= SWD_NOPWR
+
+# disabling PWM to prevent interference when PA3 remapped to ADC
+$(TARGET).CFLAGS+=-DUSE_PWM5=0
```

Comment:
not sure if it is worth having a new board file just for Delfly - maybe a better way would be to define the changes (no LEDs, USE_PWM5) in the airframe config file and use regular lisa_mxs makefile

Reply:
Yeah, I wasn't either... Discussed this with @kirkscheper and we agreed this will be the best solution since the autopilot connections on the Nimble are fixed now (and are quite different from the standard mapping). 

We could also do this in the airframe files, but this would mean a lot of copy-pasting for every new DelFly (they are all slightly different, so typically require different airframes...)

Reply:
OK, lets leave this as is.